### PR TITLE
:seedling: cleanup: remove v1 comment

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**
Remove the comment over v1, since no longer exists v1 code in the project

**Motivation**
Closes: https://github.com/kubernetes-sigs/kubebuilder/issues/1781

### CODE REVIEW

**Review by Adirio — APPROVED** [review]

**Comment by Adirio** [comment]
The error from Travis is due to a leak in #1760 which missed one of the files in `testdata`. This should probably be addressed in a different PR so that it doesn't block every PR.

File: testdata/project-v3-multigroup/controllers/apps/suite_test.go
Comment (camilamacedo86) [inline testdata/project-v3-multigroup/controllers/apps/suite_test.go:49]: See: https://github.com/kubernetes-sigs/kubebuilder/pull/1789
/hold 
until the above one gets merged it rebased.

Comment (Adirio) [reply testdata/project-v3-multigroup/controllers/apps/suite_test.go:49]: Ping me for a `lgtm`label after a rebase

**Comment by camilamacedo86** [comment]
/hold cancel

rebased wth master

**Review by Adirio — APPROVED** [review]
/lgtm

**Pull request merged**
